test/integration/apiserver/crd_validation_expressions_test.go

```diff
@@ -685,13 +687,14 @@ var structuralSchemaWithDefaultMapKeyTransitionRule = []byte(`
               "k1",
               "k2"
             ],
-            "x-kubernetes-list-type": "map",
+			"x-kubernetes-list-type": "map",
+			"maxItems": 100,
```

Comment:
nit: indentation

Reply:
Sorry, I must have inserted tabs there by mistake. Should be fixed now.